packages/blockchain-extension/tutorials/new-tutorials/basic-tutorials/a5.md

```diff
@@ -118,20 +118,22 @@ Name the file '**create.ts**'.
 In the editor view for the new create.ts file, copy and paste the following text. (The contents are also <a href="./resources/create.ts">available here</a>).
 
 ```typescript
-import { FileSystemWallet, Gateway } from 'fabric-network';
+import { Gateway, Wallets } from 'fabric-network';
 import * as path from 'path';
+import * as fs from 'fs';
 
 async function main() {
     try {
 
         // Create a new file system based wallet for managing identities.
         const walletPath = path.join(process.cwd(), 'Org1Wallet');
-        const wallet = new FileSystemWallet(walletPath);
+        const wallet = await Wallets.newFileSystemWallet(walletPath);
```

Comment:
Possibly need to change line 463 to say
`1. The wallet location on the local file system is specified using newFileSystemWallet()`
instead of:
`1. The wallet location on the local file system is specified using FileSystemWallet()` ?